chainerx_cc/chainerx/native/native_device/misc.cc

```diff
@@ -15,6 +15,42 @@ namespace chainerx {
 namespace native {
 namespace {
 
+class NativeFabsOp : public FabsOp {
+public:
+    void Call(const Array& x, const Array& out) override {
+        Device& device = x.device();
+        device.CheckDevicesCompatible(x, out);
+        const Array& x_cast = x.dtype() == out.dtype() ? x : x.AsType(out.dtype());
+        VisitFloatingPointDtype(out.dtype(), [&](auto pt) {
+            using T = typename decltype(pt)::type;
+            struct Impl {
+                void operator()(int64_t /*i*/, T x, T& out) { out = chainerx::Fabs(x); }
+            };
+            Elementwise<const T, T>(Impl{}, x_cast, out);
+        });
+    }
+};
+
+CHAINERX_REGISTER_OP_NATIVE(FabsOp, NativeFabsOp);
+
+class NativeSignOp : public SignOp {
+public:
+    void Call(const Array& x, const Array& out) override {
+        Device& device = x.device();
+        device.CheckDevicesCompatible(x, out);
+        const Array& x_cast = x.dtype() == out.dtype() ? x : x.AsType(out.dtype());
+        VisitNumericDtype(out.dtype(), [&](auto pt) {
+            using T = typename decltype(pt)::type;
+            struct Impl {
+                void operator()(int64_t /*i*/, T x, T& out) { out = chainerx::Sign(x); }
+            };
+            Elementwise<const T, T>(Impl{}, x_cast, out);
+        });
+    }
+};
+
+CHAINERX_REGISTER_OP_NATIVE(SignOp, NativeSignOp);
```

Comment:
We recently renamed the macro to `CHAINERX_NATIVE_REGISTER_OP`. Please rebase to master and fix macro names.

Reply:
I guess I'll wait till the #6944 goes in.